chartpress.py

```diff
@@ -1307,5 +1324,19 @@ def main(argv=None):
             )
 
 
+def remove_config_arg(argv):
```

Comment:
Can you make this return a copy instead of modifying the list in place? This will be easier to reason about/maintain in future. I think it'd be also worth renaming it `_remove_config_arg` since there's no reason to want to call this from outside.